ocs_ci/ocs/resources/objectconfigfile.py

```diff
@@ -135,3 +136,57 @@ def get(self, namespace=None):
         """
         out = self._run_command("get", namespace, out_yaml_format=True)
         return yaml.safe_load(out)
+
+    def describe(self, namespace=None):
```

Comment:
Wouldn't `get` object help here? describe is nothing but a simpler `get` in YAML format. 
https://stackoverflow.com/questions/57740304/kubectl-describe-vs-get-o-format

Reply:
yes, we can use it. Added this function based on the oc functions available in ocp.py since kube_job too uses similar cmds for obj creation and evaluations.
Can we continue to use this ?